app/src/main/java/org/wikipedia/feed/view/ListCardItemView.java

```diff
@@ -93,7 +94,7 @@ public ListCardItemView(Context context) {
 
     @OnClick void onClick(View view) {
         if (callback != null && entry != null && card != null) {
-            callback.onSelectPage(card, entry);
+            callback.onSelectPage(card, entry, TransitionUtil.getSharedElements(getContext(), titleView, subtitleView, imageView));
```

Comment:
Are we sure we want the `title` and `subtitle` as shared elements here? They are a different font and different size, and don't really look smooth when transitioning.

Reply:
That's a good question and I have removed them. Also I'll leave a comment on the ticket.